services/Mumble/MumblePost.ts

```diff
@@ -127,7 +123,7 @@ export class MumblePostService extends MumbleService {
         accept: 'application/json',
       },
     });
-    return responseApi as IPostItem;
+    return responseApi as any;
```

Comment:
Is that really any or can we use IPotstItem?

Reply:
no we get a raw item (without createdDateTime). 
Will type it. 👍🏽